content/workload-reduction-toolkit/identify-workload-issues/staff-workload-survey.md

```diff
@@ -15,7 +15,7 @@ resources from the workload reduction toolkit.
   <div class="govuk-grid-column-full">
     <div class="info-box">
       <div class="info-box__corner">
-        <img src="/assets/images/clipboard-icon.svg" alt="Clipboard icon">
+        <img src="/assets/images/download-icon.svg" alt="Download icon">
```

Comment:
I'm a bit confused here - the survey should be the clipboard icon, no other resources should be.